task/buildah-oci-ta/0.1/buildah-oci-ta.yaml

```diff
@@ -426,7 +426,7 @@ spec:
           add:
             - SETFCAP
     - name: sbom-syft-generate
-      image: quay.io/redhat-appstudio/syft:v0.105.1@sha256:1910b829997650c696881e5fc2fc654ddf3184c27edb1b2024e9cb2ba51ac431
+      image: registry.access.redhat.com/rh-syft-tech-preview/syft-rhel9:1.4.1@sha256:34d7065427085a31dc4949bd283c001b91794d427e1e4cdf1b21ea4faf9fee3f
```

Comment:
if syft is based on ubi9, why it's called syft-rhel9 and not syft-ubi9 ?

ubi and rhel is not the same